### ISSUE TITLE
Edge Chromium has incompatible browser & driver versions on Azure

### ISSUE THREAD

**Issue by Hexcles** [web-platform-tests/wpt#17403]
Currently, Edge Chromium isn't running any tests on Azure because:

```
SessionNotCreatedException: session not created (500): session not created: Microsoft Edge version must be 75
```

For reference, we use the latest dev channel of Edge Chromium on Azure:
https://github.com/web-platform-tests/wpt/blob/b6343fc17a3035a0c2170f44fb63df2356aa2cb9/tools/ci/azure/install_edge.yml#L2

Supposedly the problem is in `install_webdriver`:
https://github.com/web-platform-tests/wpt/blob/b6343fc17a3035a0c2170f44fb63df2356aa2cb9/tools/wpt/browser.py#L707

I'm not sure which EdgeDriver version this URL points to, but apparently the URL doesn't contain version/channel information so it's guaranteed that this won't always work.

For now, I'll be passing `--disable-build-check` to EdgeDriver (similar to #16448), assuming that the underlying version check is the same as ChromeDriver.

**Comment by Hexcles** [comment]
cc @mustjab

**Comment by foolip** [comment]
@Hexcles I've merged your quickfix, and will rename the bad runs we got on wpt.fyi so far. Before this is closed, can you check if any more trickled in?

@mustjab I'll go ahead and assign this to you, is that OK?

**Comment by foolip** [comment]
@Hexcles I've merged your quickfix, and will rename the bad runs we got on wpt.fyi so far. Before this is closed, can you check if any more trickled in?

@mustjab I'll go ahead and assign this to you, is that OK?

**Comment by foolip** [comment]
@Hexcles I've merged your quickfix, and will rename the bad runs we got on wpt.fyi so far. Before this is closed, can you check if any more trickled in?

@mustjab I'll go ahead and assign this to you, is that OK?

**Comment by foolip** [comment]
https://wpt.fyi/results/?run_id=242900005 was good again, so there aren't any more runs to rename. (I renamed them to brokenedge.)

**Comment by stephenmcgruer** [comment]
@mustjab was this fixed by https://github.com/web-platform-tests/wpt/pull/17540 ?

**Comment by mustjab** [comment]
@stephenmcgruer Yes, should be fixed now with that change and #20023 , let me know if you still see this fail.
